src/ActionModal/components/ActionModal.vue

```diff
@@ -639,7 +639,8 @@ export default {
     goToSession() {
       this.close()
 
-      this.$router.push(`/portfolio`)
+      this.$store.dispatch(`signOut`, this.network)
```

Comment:
in which case is this needed?

Reply:
this is when a user is in explore mode. if you open the action modal in explore mode you see the sign in button. so you have to sign out and then you can see the options for signing in. we used to send the user to the welcome screen but this doesn't exist anymore.